lnwallet/channel.go

```diff
@@ -1893,6 +1893,33 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 				payDesc.LogIndex)
 		}
 
+		var height uint64
+		var heightSet bool
+		if payDesc.EntryType != Add {
```

Comment:
why is it that we need to do this for `FeeUpdate`?

Reply:
We may have signed for the `FeeUpdate` just like the other ones, so we *should* set the heights or they won't be set.

Reply:
Yeah we now treat `FeeUpdate` like any other update.